base/PerfOptions.php

```diff
@@ -291,7 +295,17 @@ public function __construct(Vector<string> $argv) {
     $this->daemonOutputToFile = $this->getBool('daemon-files');
 
     $argTempDir = $this->getNullableString('temp-dir');
+    
+    if(array_key_exists('server-threads', $o)){
+      $this->serverThreads = $this->args['server-threads'];
+      $this->hhvmExtraArguments[count($this->hhvmExtraArguments)] = '-dhhvm.server.thread_count='.$this->serverThreads;
```

Comment:
`serverThreads` should be set by default to 100, and it should be removed from the config files https://github.com/hhvm/oss-performance/blob/master/conf/php.ini#L20.  Probably don't use the hhvm extra args for this, and just add another argument in https://github.com/hhvm/oss-performance/blob/master/base/HHVMDaemon.php#L70